src/Taxjar/TaxjarApi.cs

```diff
@@ -58,6 +59,7 @@ public TaxjarApi(string token, object parameters = null)
             }
 
             apiClient = new RestClient(apiUrl);
+            apiClient.UserAgent = getUserAgent();
```

Comment:
Does this still allow passing a customer user agent header?

Reply:
I believe it was possible to overwrite our custom user agent using `SetApiConfig` in an earlier commit. However, similar to our other API clients this is no longer possible since we add the user agent for each request.